pages/installation/operating_system_packages.rst

```diff
@@ -57,6 +57,19 @@ Debian 7               SysV        ``sudo update-rc.d graylog-server defaults 95
 Debian 8, Ubuntu 16.06 systemd     ``sudo systemctl enable graylog-server``
 ====================== =========== ==================================================
 
+Manual Repository Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't like to install the repository DEB to get the repository configuration onto your system, you can do so manually (although we don't recommend to do that).
+
+First, add the `Graylog GPG key <https://github.com/Graylog2/fpm-recipes/blob/master/recipes/graylog-repository/files/deb/graylog-keyring.gpg>`_ which is being used to sign the packages to your system.
```

Comment:
We should probably use the GPG key from our packages service here: https://packages.graylog2.org/repo/debian/keyring.gpg